web/html/src/utils/data-providers/simple-data-provider.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+
+import PageControl from './page-control';
+
+type PagedData = {
+  items: Array<any>,
+  total: number
+};
+
+export default class SimpleDataProvider {
+  data: Array<any>;
+  identifier: (row: any) => any;
+  filter: ?(row: any, criteria: string) => boolean;
+  comparators: ?{column: (a: any, b: any, key: string, direction: number) => number};
```

Comment:
Would be easier to read with a `ComparatorType`. And that type would be reused further down too

Reply:
:+1: